applications/luci-app-adbyby-plus/root/etc/init.d/adbyby

```diff
@@ -3,13 +3,10 @@
 START=96
 STOP=10
 
-extra_command "add_rule"
```

Comment:
keep it as-is

Reply:
Why, it had new variable call `EXTRA_COMMANDS`

Reply:
Fixed!